tests/framework/helpers/HtmlTest.php

```diff
@@ -100,6 +101,68 @@ public function testJsFile()
         $this->assertEquals("<!--[if (gte IE 9)|(!IE)]><!-->\n" . '<script src="http://example.com"></script>' . "\n<!--<![endif]-->", Html::jsFile('http://example.com', ['condition' => '(gte IE 9)|(!IE)']));
     }
 
+    public function testCsrfMetaTagsDisableCsrfValidation()
+    {
+        $this->assertEquals('', Html::csrfMetaTags());
```

Comment:
This is test coverage for framework/helpers/BaseHtml.php:305 result if Disable Csrf Validation:

https://scrutinizer-ci.com/g/yiisoft/yii2/code-structure/master/code-coverage/framework/helpers/BaseHtml.php